src/sql/workbench/contrib/notebook/browser/markdownToolbarActions.ts

```diff
@@ -334,6 +349,13 @@ export class MarkdownTextTransformer {
 				endColumn: this.getEndTextToInsert(type).length + 1,
 				endLineNumber: selection.endLineNumber + 1
 			});
+		} else if (lineType === MarkdownLineType.EVERY_LINE) {
```

Comment:
If we do
```
if
else
else if
```

Does the code in the `else if` get executed if the condition is true?

If not, then this condition isn't doing anything, and can be removed.

If yes, please move it before the `else`.

Reply:
Fixed!